server/graphql/purchase_order/src/purchase_order_queries.rs

```diff
@@ -142,6 +142,7 @@ impl PurchaseOrderFilterInput {
                 .map(|t| map_filter!(t, |s| PurchaseOrderStatus::from(s))),
             supplier: self.supplier.map(StringFilter::from),
             store_id: self.store_id.map(EqualFilter::from),
+            created_datetime: self.created_datetime.map(DatetimeFilter::from),
```

Comment:
The GraphQL layer had a "createdDatetime" filter input, but it wasn't actually being passed through to repository layer, so was just giving incorrect results